packages/core/IUtils.d.ts

```diff
@@ -59,6 +59,17 @@ export interface DateIOFormats<TLibFormatToken = string> {
   keyboardDateTime24h: TLibFormatToken;
 }
 
+export type Unit =
```

Comment:
Form of the names better to be in the plural form. “years”, “seconds”, etc

Reply:
just added plural forms of units